examples/linear_model/plot_ols_ridge_variance.py

```diff
@@ -47,25 +47,22 @@
     fig = plt.figure(fignum, figsize=(4, 3))
     plt.clf()
```

Comment:
Ok it doesn't really make a difference whether `plt.clf()` is used or not (tested both on notebook and python 3.6). I presume it might make a difference when using older versions of pyplot, but that be hard to test since I don't know which version the original tutorial was written in.

Reply:
Hmm, I guess it's useless because it does not occur very often, but let's keep it here.